l10n_es_aeat_sii_oca/models/account_move.py

```diff
@@ -810,6 +811,21 @@ def _compute_sii_enabled(self):
             else:
                 invoice.sii_enabled = False
 
+    @api.model
+    def _is_unsupported_search_operator(self, operator):
+        return operator != "="
```

Comment:
Creo que los operadores negativos tampoco valdrían, porque el dominio debe ser el inverso.

Reply:
El filtro definido en la vista `('sii_enabled', ''=', True)` funciona, veo que al filtrar por ese campo se intenta aplicar `!= True`, por lo que creo que es mejor quitar directamente este método.

Reply:
No, bueno, será poner `return operator not in ("=", "!=")` si dices que funciona, pero la restricción hay que dejarla para no obtener resultados incorrectos. E insisto que creo que no funciona, porque hay que dar la vuelta a todos los and y or.

Reply:
Vamos, que entonces está así correcto.

Reply:
Restituido el método con la condición que indicas y añadido un pequeño test para probar el search con el campo en cuestión.